.github/ISSUE_TEMPLATE/funding_application.yml

```
@@ -0,0 +1,220 @@
+---
+name: TI Funding Request Form
+description: TI Funding Request Form OpenSSF
+title: "[Technical Initiative Funding Request]: "
+labels:
+  - administration
+  - For Review
+  - Funding Request
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this Funding Request.
+  - type: markdown
+    attributes:
+      value: |
+        **Pre-conditions:**
+        - The TI must have an identified lifecycle phase inside the OpenSSF, such as sandbox, incubating, or graduated. 
+        - The TI requesters should know 
+          - Within general funding limits for the TIs at the respective lifecycle, that are to be determined. 
+          - Within approved categories of asks (e.g. no asking for a TI outing at a fancy restaurant but yes to cloud credits)
+  - type: markdown
+    attributes:
+      value: |
+        **Workflow:**
+        1. TI or TIs working together see a need for funding some work that doesn’t exist within an existing give/get that is pre-approved. For example cloud credits to do a PoC or case study.
+        2. TI(s) get consensus on the scope of how much money they are asking for and to be spent on what. For example $10k-$15k in cloud credits over a year to spend on building out a PoC to test out <some set of things>
+        3. TI(s) drafts that includes:
+          - Problem Statement in the form below
+          - Proposed work and other work considered
+          - Benefit to OSSF and ecosystem
+          - Approximate cost
+          - If a third party (like a cloud provider) is requested, please list any vendor preferences (Note, these are not guaranteed and if the request is approved, you will work with OpenSSF Staff to finalize vendors to reach alignment where possible).
+        4. TI submits this issue to the TAC. This will be treated like a CFP. Respondents will be notified in writing, and approved proposals will be published.
+        5.  Submitted funding requests will be batched and reviewed quarterly, relative to each other. Dates where proposals can be accepted, as well as when the review cycle will occur, and when decisions will be made. The TAC reserves a weeklong window for technical merit, and the finance review will occur the following week.
```

```suggestion
        5.  Submitted funding requests will be reviewed at least quarterly, relative to each other. Dates where proposals can be accepted, as well as when the review cycle will occur, and when decisions will be made. The TAC reserves a weeklong window for technical merit, and the finance review will occur the following week.
```

This is a minor suggestion / feel free to ignore or iterate on it further. The idea here is that this creates a window for us to fast-track a request for funding if we really need to, while still keeping the general quarterly cycle.